readthedocs/projects/forms.py

```diff
@@ -357,6 +358,10 @@ def clean_conf_py_file(self):
             )  # yapf: disable
         return filename
 
+    def clean_build_config_file(self):
+        filename = self.cleaned_data.get("build_config_file", "").strip()
+        return filename
+
```

Comment:
Is there something we'd want to validate here? Should we disallow leading `/` for some consistency?

Reply:
Should definitely disallow trailing `/`

Wondering what's a good utility function here for file paths...